scene/animation/animation_blend_tree.h

```diff
@@ -157,6 +164,8 @@ class AnimationNodeOneShot : public AnimationNodeSync {
 	StringName fade_out_remaining = "fade_out_remaining";
 	StringName time_to_restart = "time_to_restart";
 
+	void _maybe_notify(bool p_active, bool p_internal_active, double p_fade_in_remaining);
```

Comment:
```suggestion
	void _check_notify(bool p_active, bool p_internal_active, double p_fade_in_remaining);
```
Indicate what exactly the function is doing.